src/pages/signup-affiliates/index.tsx

```diff
@@ -1,11 +1,206 @@
-import { WithIntl } from 'components/localization'
+import { localize, WithIntl } from 'components/localization'
+import { Input, Button } from 'components/form'
+import { Header, LinkText, LocalizedLinkText } from 'components/elements'
+import device from 'themes/device'
+import Google from 'images/svg/custom/googlenew.svg'
+import Apple from 'images/svg/custom/applenew.svg'
+import Facebook from 'images/svg/custom/facebooknew-blue.svg'
 
-const AffiliateSignup = () => {
+const StyledNote = styled.div`
+    padding: 8px 0;
+    margin: 24px 0 16px;
+    height: 36px;
+    width: 406px;
+    border-radius: 4px;
+    background-color: rgba(248, 248, 249, 1);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: flex-start;
+
+    @media ${device.mobileL} {
+        height: 54px;
+        padding: 8px 7px;
+        width: 100%;
+    }
+`
+const StyledLinkText = styled(LinkText)`
+    margin-left: 3px;
+    font-size: 14px;
+    line-height: 20px;
+    height: 20px;
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+const InputGroup = styled.div`
+    width: 100%;
+    margin-bottom: 16px;
+`
+const EmailButton = styled(Button)`
+    width: 100%;
+    font-size: 1.4rem;
+    margin-bottom: 0.4rem;
+    margin-top: 3.2rem;
+
+    @media ${device.tabletL} {
+        margin-top: 24px;
+    }
+
+    @media ${device.mobileL} {
+        font-size: 1.75rem;
+    }
+`
+const StyledText = styled(LinkText)`
+    font-size: 14px;
+    line-height: 20px;
+    @media (min-width: 768px) {
+        width: 14rem;
+    }
+    @media (min-width: 820px) and (max-width: 1200px) {
+        width: 17rem;
+    }
+`
+const LoginText = styled(LinkText)`
+    text-align: center;
+    align-self: center;
+    margin-top: 1.6rem;
+
+    @media ${device.tabletL} {
+        font-size: 2rem;
+    }
+`
+const Line = styled.div`
+    width: 132px;
+    height: 1px;
+    background-color: var(--color-grey-7);
+`
+const SignupWithContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 2.4rem;
+
+    @media ${device.tabletL} {
+        margin-top: 4rem;
+    }
+`
+const LoginContainer = styled.div`
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 2.4rem;
+`
+const CaptchaContainer = styled.div`
+    margin: auto;
+`
+const SocialWrapper = styled.div`
+    width: 100%;
+    margin-top: 2.4rem;
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+`
+const GoogleWrapper = styled.div`
+    padding-right: 24px;
+`
+const AppleWrapper = styled.div`
+    padding-left: 24px;
+`
+const FacebookWrapper = styled.div``
+
+const onChange = (value) => {
+    console.log('Captcha value:', value)
+}
+type AffiliateSignupProps = {
+    autofocus: boolean
+    email: string
+    email_error_msg: string
+    is_ppc: boolean
```

Comment:
if `is_ppc` is not used we can now remove it.

Reply:
okay